src/main/resources/changelog/changes/v3.2/create_instance_classification_table.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.20.xsd">
+
+  <changeSet id="MSEARCH-667@@create_instance_classification_table" author="Pavlo_Smahin">
+    <preConditions>
+      <not>
+        <tableExists tableName="instance_classification"/>
+      </not>
+    </preConditions>
+
+    <comment>Create instance_classification table</comment>
+
+    <createTable tableName="instance_classification">
+      <column name="classification_type" type="VARCHAR(255)">
+        <constraints primaryKey="true" primaryKeyName="pk_instance_classification"/>
+      </column>
+      <column name="classification_number" type="VARCHAR(255)">
+        <constraints primaryKey="true" primaryKeyName="pk_instance_classification"/>
+      </column>
+      <column name="tenant_id" type="VARCHAR(255)">
+        <constraints primaryKey="true" primaryKeyName="pk_instance_classification"/>
+      </column>
+      <column name="instance_id" type="VARCHAR(255)">
```

Comment:
Why VARCHAR(250)?

Reply:
It can be reduced to size of UUID. Not using of UUID type caused by wish to reduce amount of operations (converting UUID to String, String to UUID in java code and casting text to uuid on database side) and following approach of consortium_instance table.